currency_rate_update_VN_VCB/__manifest__.py

```diff
@@ -0,0 +1,17 @@
+# © 2018 Komit <http://komit-consulting.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+{
+    'name': 'Currency Rate Update - VCB',
+    'summary': 'Currency Rate Update Vietcombank service',
+    'version': '11.0.1.0.0',
+    "author": "Komit, Odoo Community Association (OCA)",
+    "website": "http://komit-consulting.com",
+    "license": "AGPL-3",
+    "category": "Financial Management/Configuration",
+    "depends": [
+        "base",
```

Comment:
I think `currency_rate_update` already depends on `base`, so you don't need to depends on it